geotrek/signage/management/commands/loadsignage.py

```diff
@@ -178,7 +178,13 @@ def handle(self, *args, **options):
 
                     structure = Structure.objects.get(name=feature.get(field_structure_type)) if field_structure_type in available_fields else structure
                     description = feature.get(field_description) if field_description in available_fields else default_description
-                    year = int(feature.get(field_implantation_year)) if field_implantation_year in available_fields and feature.get(field_implantation_year).isdigit() else default_year
+
+                    year = feature.get(field_implantation_year) if field_implantation_year in available_fields else default_year
+                    if year:
+                        year = int(year) if str(year).isdigit() else default_year
```

Comment:
Maybe we should enforce that the `year` values are expected to be integers or null and fail with an error in the other cases or skip the feature with a warning. What do you think @submarcos @camillemonchicourt ?

Reply:
With this change non-digit textual values would be silently changed into the default year value.

Reply:
Yes we should accept only integers or null values

Reply:
fixed in 09b13b8